examples/react-router/cosmos.proxies.js

```diff
@@ -0,0 +1,3 @@
+import createReactRouterProxy from 'react-cosmos-router-proxy';
+
+export default [createReactRouterProxy()];
```

Comment:
As a user I would love to see an example with multiple proxies.

Reply:
Good point, will add.